src/submission/forms.py

```diff
@@ -82,13 +82,13 @@ class AuthorForm(forms.ModelForm):
 
 	class Meta:
```

Comment:
Excluding sequence from Author and Editor forms because it wasn't displayed on forms and would reset to null every time the form was saved